### PR TITLE
Operator migration supported only on the same Calico version

### PR DESCRIPTION
**Pull request by MichalFupso**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

**Review by mgleung — APPROVED** [review]

File: calico/operations/operator-migration.mdx [hunk 1]
```diff
@@ -42,6 +42,7 @@ and calculate how to take ownership of them. The operator will maintain existing
 ## Before you begin
 
 - Ensure that your {{prodname}} installation is configured to use the Kubernetes datastore. If your cluster uses etcdv3 directly, you must follow [the datastore migration procedure](datastore-migration.mdx) before following this document.
+- Migration to operator-managed installation is only supported between the same {{prodname}} versions
```
Comment (bmckercher123) [inline calico/operations/operator-migration.mdx]:
```suggestion
- Migration to operator-managed installation is supported only between the same {{prodname}} versions
```

**Review by bmckercher123 — APPROVED** [review]
Small fix to all files. Will merge.

**Review by bmckercher123 — APPROVED** [review]
Approved

**Comment by ctauchen** [comment]
I noticed that this commit didn't get picked up in 3.26: https://docs.tigera.io/calico/latest/operations/operator-migration#before-you-begin

@girishshankaran I suspect the 3.26 publishing PR wasn't quite up to date. Can you fix this and check for all changes to Next that occurred between your PR's latest rebase and publishing?

FYI @bmckercher123

**Pull request merged**